circle.yml

```diff
@@ -1,35 +1,84 @@
+defaults: &defaults
```

Comment:
I think that according to their docs it should be under `./circleci/config.yml`

Reply:
`./.circleci/config.yml`

Reply:
done